content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-discovery-onboarding/index.md

```diff
@@ -57,10 +57,27 @@ Chainguard Enforce Discovery exposes an API that will return metadata about clus
 * `eligible`: this cluster is ready to enroll with Chainguard Enforce.
 * `enrolled`: this cluster is already enrolled in Enforce.
 
-There are currently two options for how you can use Discovery: using the `chainctl cluster discover` command or the Chainguard Terraform provider.
+In order for a cluster to be eligible, it must either have a public endpoint with no access control or, if it does have access control, it must give Chainguard Enforce access. If a cluster isn't supported, it's typically because of locked down control planes; for example, bastion or control plane authorized networks on GKE.
```

Comment:
Do you think "give Chainguard access" may be more correct here? I am not sure if we should be thinking of it as the company entity or as the product entity. This is a bit in the weeds though, so don't get blocked on this.